src/components/13-collection/collection--fancy-date.njk

```diff
@@ -0,0 +1,188 @@
+          </time>
+        </a>
+        <div class="usa-collection__body">
+          <h3 class="usa-collection__heading">
+            <a class="usa-link" href="https://www.performance.gov/presidents-winners-press-release/">Gears of Government President's Award Winners</a>
+          </h3>
+          <p class="usa-collection__teaser">Today, the Administration announces the winners of the Gears of Government President’s Award. This program recognizes the contributions of individuals and teams across the federal workforce who make a profound difference in the lives of the American people.</p>
+          <div class="usa-collection__meta-list">
+            <span class="usa-collection__meta-item">
+              By Performance.gov Team
+            </span>
+            <span class="usa-collection__meta-item">
+              <time datetime="2020-09-30T12:00:00+01:00">
+                September 30, 2020
+              </time>
+            </span>
+          </div>
+
+          <div class="usa-collection__meta-list">
+            <span class="usa-collection__meta-item usa-tag">
+              New
+            </span>
+            <span class="usa-collection__meta-item usa-tag">
+              PMA
+            </span>
+            <span class="usa-collection__meta-item usa-tag">
+              OMB
+            </span>
+          </div>
+        </div>
+      </li>
+      <li class="usa-collection__item">
+        <a class="usa-collection__calendar-date" href="https://www.performance.gov/sba-wosb-dashboard/">
+          <time datetime="2020-09-30T12:00:00+01:00">
+            <span class="usa-collection__calendar-date-month">SEP</span>
+            <span class="usa-collection__calendar-date-day">30</span>
+          </time>
+        </a>
+        <div class="usa-collection__body">
+          <h3 class="usa-collection__heading">
+            <a class="usa-link" href="https://www.performance.gov/sba-wosb-dashboard/">Women-Owned Small Business Dashboard</a>
+          </h3>
+          <p class="usa-collection__teaser">In honor of National Women’s Small Business Month, we’ve partnered with SBA’s Office of Government Contracting and Business Development and Office of Program Performance, Analysis, and Evaluation to highlight the Women-Owned Small Businesses (WOSBs) data dashboard!</p>
+          <div class="usa-collection__meta-list">
+            <span class="usa-collection__meta-item">
+              By Small Business Administration (SBA) and Performance.gov Team
+            </span>
+            <span class="usa-collection__meta-item">
+              <time datetime="2020-09-30T12:00:00+01:00">
+                September 30, 2020
+              </time>
+            </span>
+          </div>
+
+          <div class="usa-collection__meta-list">
+            <span class="usa-collection__meta-item usa-tag">
+              SBA
+            </span>
+            <span class="usa-collection__meta-item usa-tag">
+              Beta data
```

Comment:
Should this say "Meta data"?

Reply:
That was the content from the examples but I was wondering the same so I'll update it. Thanks!